notebooks/demo_processes.ipynb

```diff
@@ -0,0 +1,622 @@
+    "hydromodel_climos_opendap = \"http://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets/TestData/tiny_hydromodel_gcm_climos.nc\"\n",
+    "\n",
+    "output = thunderbird.split_merged_climos([tasmax_climos_local, hydromodel_climos_opendap])\n",
+    "req = requests.get(output.get()[0])\n",
+    "print(BeautifulSoup(req.content.decode('utf-8')))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Update Metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This process runs [update_metadata](https://github.com/pacificclimate/climate-explorer-data-prep/blob/master/README.md#update_metadata-update-metadata-in-a-netcdf-file), which updates metadata from an input file according to instructions provided by a string or `.yaml` file. Rather than a metalink, the output is the updated netcdf file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\u001b[0;31mSignature:\u001b[0m \u001b[0mthunderbird\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mupdate_metadata\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnetcdf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mupdates\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mNone\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+       "\u001b[0;31mDocstring:\u001b[0m\n",
+       "Update file containing missing, invalid, or incorrectly named global or variable metadata attributes\n",
+       "\n",
+       "Parameters\n",
+       "----------\n",
+       "netcdf : ComplexData:mimetype:`application/x-netcdf`, :mimetype:`application/x-ogc-dods`\n",
+       "    NetCDF file\n",
+       "updates : string\n",
+       "    The filepath of an updates file that specifies what to do to the metadata it finds in the NetCDF file\n",
+       "\n",
+       "Returns\n",
+       "-------\n",
+       "output : ComplexData:mimetype:`application/x-netcdf`\n",
+       "    Output Netcdf Files\n",
+       "\u001b[0;31mFile:\u001b[0m      ~/thunderbird/notebooks/</home/slim/thunderbird-venv/lib/python3.6/site-packages/birdy/client/base.py-2>\n",
+       "\u001b[0;31mType:\u001b[0m      method\n"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Check info on `update_metadata` process\n",
+    "thunderbird.update_metadata?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "update_metadataResponse(\n",
+      "    output='http://localhost:5001/outputs/87b92afc-b663-11ea-afd0-c86000e3f2fd/gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100_copy.nc'\n",
+      ")\n"
+     ]
+    }
+   ],
+   "source": [
+    "updates = resource_filename(\"tests\", \"metadata-conversion/simple-conversion.yaml\")\n",
```

Comment:
I think this should be `data/metadata-conversion/simple-conversion.yaml`?

Reply:
According to master, the path is `tests/metadata-conversion/simple-conversion.yaml`.

Reply:
Also, I don't see `data` directory` in `venv/lib/python3.7/site-packages/tests` a virtual environment is set. @nikola-rados , do you have any idea on this?

Reply:
That should be pointing to `thunderbird/tests/`...

Reply:
Hmm not sure @sum1lim I can see the data dir.

Reply:
Do your pytests work?

Reply:
Never mind `resource_filename("tests", "_")` changes depending on if your current directory is `notebooks` or `thunderbird`.